### ISSUE TITLE
Outdated node version

### ISSUE THREAD

**Issue by hbruch** [HSLdevcom/digitransit-ui#4520]
Digitransit-UI currently uses outdated node 10. Even for node 12, maintenance phase (=security support) ends in about 5 weeks.

To explore what needs to be done, we started [branch node12](https://github.com/stadtnavi/digitransit-ui/tree/node12)

To at least get it started, we had to remove `node_modules/@babel/runtime/helpers/esm/package.json`, otherwise we encountered a ERR_REQUIRE_ESM. (There is hopefully better solution that that)

Does anyone have further suggestions to move forward?

**Comment by vesameskanen** [comment]
Updating nodejs version is indeed badly needed. It seems that the update is not possible unless we update many npm libraries too and those updates involve breaking changes. One example is react-leaflet; the new map api looks quite different and adopting it will be a challenging task.

Our resources are currently tied to other issues, and we cannot start nodejs update before the autumn. However, if you find it possible to start the task in  the near future, we will do our best to help in the process.

**Comment by derhuerst** [comment]
As of #4607, `digitransit-ui` now uses Node 16.

**Comment by derhuerst** [comment]
I forgot to mention that this only applies to the `node-migration` branch for now, but the goal is to get it merged into `v2`.
